.vscode/schema.json

```diff
@@ -71,6 +71,7 @@
     "definitions": {
         "Tab": {
             "type": "object",
+            "additionalProperties": false,
```

Comment:
y can do far more fore validation // auto-completion inside json schema.
i u want, i can PR u some exemple

Reply:
What do you mean?
I added `additionalProperties` since it will invalidate the JSON if adding properties not specified by the schema